### PR TITLE
Fix running the checks on a directory with whitespace in its path

### PR DESCRIPTION
**Pull request by theMarix**

### CODE REVIEW

**Review by ddiss — CHANGES_REQUESTED** [review]
Looks like obvious goodness to me. @theMarix please rebase and then I'll try to track down a maintainer to get this merged. Thanks!

**Comment by theMarix** [comment]
Thanks for picking this up, @ddiss . Rebase is done.

File: 45-stale-changes [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 test "$1" = "--verbose" && { VERBOSE=true ; shift ; }
 test "$1" = "--batchmode" && { BATCHMODE=true ; shift ; }
-DIR_TO_CHECK=$1
+DIR_TO_CHECK="$1"
 DESTINATIONDIR=$2
```
Comment (ddiss) [inline 45-stale-changes:6]: Thanks for the respin - should the `DESTINATIONDIR` assignment be quoted here too?

File: 70-baselibs [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 test "$1" = "--verbose" && { VERBOSE=true ; shift ; }
 test "$1" = "--batchmode" && { BATCHMODE=true ; shift ; }
-DIR_TO_CHECK=$1
+DIR_TO_CHECK="$1"
 DESTINATIONDIR=$2
```
Comment (ddiss) [inline 70-baselibs:6]: Same for above

File: 45-stale-changes (hunk 1 above)
Comment (theMarix) [reply 45-stale-changes]: In my tests this did not show up as an issue. But simply for consistency we should. And maybe somebody has an even more weird set-up than mine so it'll actually help.

**Review by ddiss — APPROVED** [review]
Looks good - thanks again for your contribution!
@adrianschroeter which maintainer should I ping to get this merged?

**Comment by ddiss** [comment]
> @adrianschroeter which maintainer should I ping to get this merged?

Ping - @sjamgade @bugfinder anyone else able to merge this?

**Pull request merged**